TreeTracker/Utilities/Validation.swift

```diff
@@ -75,12 +81,4 @@ private extension Validation {
         return false
     }
 
-    static func cleanedPhoneNumber(phoneNumber: String) -> String {
```

Comment:
Please can you revert this change. It could cause invalid phone numbers to be entered.

We can just apply `.replacingOccurrences(of: " ", with: "")` in the view model for this change

Reply:
I added the below codes back to the func cleanedPhoneNumber()
             .replacingOccurrences(of: "+", with: "")
             .replacingOccurrences(of: " ", with: "")
             .replacingOccurrences(of: "(", with: "")
             .replacingOccurrences(of: ")", with: "")
             .replacingOccurrences(of: "-", with: "")

Reply:
Please could you also move it back into the private extension